### ISSUE TITLE
Footer

### ISSUE THREAD

**Issue by skleinfeldt** [PSUEducationalEquity/plonetheme.eeq#8]
As a user I can see a themed footer throughout the site, which is based on the site's footer portlets. As an editor I can change the content of the footer by editing the footer portlets. The bottom-most footer information (logo, Site Map, Accessibility etc.) can be hardcoded in the theme. Copyright year should be updated automatically.

**Comment by alecpm** [comment]
Similar to the header, this needs a couple more things:

* The Site Map, Accessibility, ... links should be pulled from the site actions (with the contact link dropped since we're putting that into the header)
* The login button should pull in the #personaltools-login login tag.
* The footer portlet titles should probably come from the portlets, if possible, rather than being hardcoded.

**Comment by silviot** [comment]
> The footer portlet titles should probably come from the portlets, if possible, rather than being hardcoded.

Is it really worth it? In case the title of the portles are changed, they will no longer show up in the footer, since the rules locate the portlets one by one using a class derived from their name. Or maybe I'm missing something.

**Comment by silviot** [comment]
@alecpm I pushed fixes to these issues to #41 together with the header fixes, since I needed `portal_url`.

I'm not sure about the problem with the color of the footer links. For me they are always white.

**Comment by alecpm** [comment]
@silviot I should have updated my comment, at some point the links all went white for me so I must have had some temporary issue with my local theme install.

You're probably right that allowing TTW portlet header customization is probably not worthwhile in the first go at this given the selection process. I would have been inclined to select and place the the first 5 portlets in the manager rather than trying to look up specific portlets by class/id. This approach is fine though.

**Comment by silviot** [comment]
The links that use `portal_url` are not working properly: _Site Map_, _Accessibility_ and _Contact_ do not include the `/web` part on https://beta.equity.psu.edu/web

Also the login/logout button is not working.

**Comment by paulrentschler** [comment]
@silviot I had to switch to another theme and then back to this theme in the Theming control panel and now the `portal_url` stuff is working as is the login/logout button.

**Comment by silviot** [comment]
@paulrentschler great! Now that you mention it, I remembered I needed to do the same locally to make it work, but then I forgot to document this necessary step.
Thanks for finding the issue!

**Comment by alecpm** [comment]
Yes, we need to be careful about deploying code with changes to the variables in `manifest.cfg`. It should be possible to make a registry (?) upgrade step for such changes.
